documentation/modules/overview/con-kafka-concepts-components.adoc

```diff
@@ -32,3 +32,4 @@ Kafka MirrorMaker:: Kafka MirrorMaker replicates data between two Kafka clusters
 Using producers and consumers, MirrorMaker consumes messages from the (source) primary cluster and writes them to the (target) secondary cluster.
 Kafka Bridge:: Kafka Bridge provides an API for integrating HTTP-based clients with a Kafka cluster.
 Kafka Exporter:: Kafka Exporter extracts data for analysis as Prometheus metrics, primarily data relating to offsets, consumer groups, consumer lag and topics. Consumer lag is the delay between the last message written to a partition and the message currently being picked up from that partition by a consumer
+Cruise Control:: Cruise Control monitors and balances workloads across a Kafka cluster.
```

Comment:
Again I thing "rebalances workloads" is misleading. We are rebalancing partitions which will change the pattern of workload on the various cluster pods.